web-server/v0.4/src/components/SiderMenu/index.js

```diff
@@ -9,9 +9,9 @@ const SiderMenuWrapper = props => {
     <DrawerMenu
       getContainer={null}
       level={null}
-      handleChild={<i className="drawer-handle-icon" />}
+      handlechild={<i className="drawer-handle-icon" />}
       onHandleClick={() => {
-        props.onCollapse(!collapsed);
+        props.onCollapse(collapsed);
```

Comment:
Can you add a comment for why this is "NOT collapsed"?

Reply:
There were certain inconsistencies in our code regarding props passed from BasicLayout on to the SiderMenu. I have made some minor refactors to it which makes it easy to predict the collapse behaviour now!